Library/Homebrew/shims/linux/super/swift

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# Ensure we use Swift's clang when performing Swift builds.
+
+export CC="clang"
+export CXX="clang++"
+export HOMEBREW_CC="swift_clang"
```

Comment:
Do we need a new symlink to this?

Reply:
Not the way it's currently implemented. We may want to change it for consistency's sake though. 

This exception is easy to understand now, but I can already see myself being confused about this the next time I look at my this code...

Reply:
> This exception is easy to understand now, but I can already see myself being confused about this the next time I look at my this code...

Agreed. I found this confusing.

Reply:
The reason a `swift_clang` symlink wouldn't ever be used is because CC and CXX stays as `clang` here.

The reason it's necessary for `llvm_clang` is because that's currently integrated into `CompilerSelector` as being a different compiler altogether than regular clang, so CC and CXX gets set to `llvm_clang`.

Reply:
Gotcha, thanks @Bo98. No code changes necessary but can you add an inline comment to the Ruby explaining this? Thanks!